src/features/isanteplus/addPatient.feature

```diff
@@ -24,5 +24,4 @@ Feature: Adding a Patient
     |sharif1   |sharif1  |Male   |10  |STCODE3 |HAITI101 | haiti   |
     |moses3    |moses3   |Male   |17  |STCODE55|HAITI103 | haiti   |
     |sharif2   |sharif2  |Male   |12  |STCODE4 |HAITI102 | haiti   |
-    |cliff     |gita     |Male   |13  |STCODE5 |HAITI104 | haiti   |
```

Comment:
You can also make this name available, Also if you use any name , its better to run it once for the system to be able to allocate itself once